app/views/layouts/_top_menu.html.erb

```diff
@@ -18,6 +18,7 @@
             <ul class="split-dropdown dropdown-menu">
               <%= nav_link "Applicants", applicants_people_path %>
               <%= nav_link "Inactive", inactive_people_path %>
+              <%= nav_link "Vendors", vendors_path if can? :read, Vendor %>
```

Comment:
This should not be under people, as it's not about the members. We'll need to create a submenu under inventory, similar to what we have now for people and events.

Reply:
done!